packages/ui/src/views/repo/pull-request/components/pull-request-diff-viewer.tsx

```diff
@@ -442,11 +442,7 @@ const PullRequestDiffViewer = ({
                       commentId={parent.id}
                       setHideReplyHere={state => toggleReplyBox(state, parent?.id)}
                       onQuoteReply={handleQuoteReply}
-                      icon={
-                        <Avatar.Root>
-                          <Avatar.Fallback>{parentInitials}</Avatar.Fallback>
-                        </Avatar.Root>
-                      }
+                      icon={<Avatar name={parentInitials} size="default" rounded />}
```

Comment:
Looks like we need the name, not the initials here and `size` isn't required.

```suggestion
                      icon={<Avatar name={parentInitials} rounded />}
```